node/pkg/node/options.go

```diff
@@ -52,6 +52,11 @@ func GuardianOptionP2P(p2pKey libp2p_crypto.PrivKey, networkId string, bootstrap
 				components.SignedHeartbeatLogLevel = zapcore.InfoLevel
 			}
 
+			// Add the gossip advertisement address if it was specified
+			if gossipAdvertiseAddress != "" {
```

Comment:
Looks like you could always just do this since the field in the component defaults to the empty string anyway.

Reply:
Fixed in e306a743d5244fe03d59c65f39a923cc1bd376a9